index.html

```diff
@@ -34,168 +41,178 @@ <h2 class="support-text">Software Developer</h2>
+                            <li><img src="./image/Ruby.svg" alt="ruby-icon"></li>
+                        </ul>
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
                 </div>
-            </section>
-            <section>
-                <img src="./image/Img-Placeholder.svg" alt="image placeholder" class="image-placeholder">
-                <div>
-                    <h2 class="multi-post-header">Multi-Post Stories</h2>
-                    <p class="multi-post-paragraph">A daily section of privately personalized reads; no accounts
-                        or sign-ups required. has been the industry&#x27;s standard
-                        dummy text ever since the 1500s, when an unknown printer took a 
-                        standard dummy text.
-                    </p>
+                <div class="multi-post">
+                    <div class="image-placeholder"></div>
+                    <div class="flex-container ">
+                        <h2 class="multi-post-header">Multi-Post Stories</h2>
+                        <p class="multi-post-paragraph">A daily section of privately personalized reads; no accounts
+                            or sign-ups required. has been the industry&#x27;s standard
+                            dummy text ever since the 1500s, when an unknown printer took a 
+                            standard dummy text.
+                        </p>
 
-                    <ul class="icon">
-                        <li><img src="./image/css.svg" alt="css-icon"></li>
-                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
-                        <li><img src="./image/html.svg" alt="html-icon"></li>
-                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
-                        <li><img src="./image/bootstrap.svg" alt="bootstrap-icon"></li>
-                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
-                        <li><img src="./image/Ruby.svg" alt="ruby-icon"></li>
-                    </ul>
-                
-                    <button type="button" class="see-project">See Project</button>
+                        <ul class="icon">
+                            <li><img src="./image/css.svg" alt="css-icon"></li>
+                            <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                            <li><img src="./image/html.svg" alt="html-icon"></li>
+                            <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                            <li><img src="./image/bootstrap.svg" alt="bootstrap-icon"></li>
+                            <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                            <li><img src="./image/Ruby.svg" alt="ruby-icon"></li>
+                        </ul>
+                        <button type="button" class="see-project">See Project</button>
```

Comment:
- [ ] Please also add hover and focus styles to all the buttons as given in the Figma design. 
![Buttons](https://user-images.githubusercontent.com/69011963/172994895-042b6934-ce5c-423d-b8aa-d8bb7eb0144c.png)


- [ ] [OPTIONAL] For accessibility concerns cursor should change to pointer 👆🏼 when you move the cursor over to the button. You can fix it by adding `cursor: pointer;` to the buttons.